package.json

```diff
@@ -63,7 +63,7 @@
     "browserify": "10.2.6",
     "browserify-shim": "3.8.9",
     "del": "1.2.0",
-    "ember-cli": "1.13.1",
+    "ember-cli": "1.13.8",
```

Comment:
Yes, there is a siginifcant speed improvement on rebuild in 1.13.8. I'm seeing between 1 and 3 seconds rebuild time improvment here. Which saves time both for test and dev. And quite a few other fixes (https://github.com/ember-cli/ember-cli/blob/master/CHANGELOG.md) 

```Brocfile.js``` is removed as stated in the transition document for ember-cli  (https://github.com/ember-cli/ember-cli/blob/master/TRANSITION.md#brocfile-transition). It could have been removed in 1.13.0, but the deprecation message starts from 1.13.5.